examples/dashboard_example.cpp

```diff
@@ -96,6 +97,8 @@ int main(int argc, char* argv[])
     return 1;
   }
 
+  sleep(1);
```

Comment:
Do we need this sleep?

Reply:
That's effectively what makes this non-flaky. Without that, the play command seems to be failing quite regularly in ci.